deepchecks/vision/simple_classification_data.py

```diff
@@ -22,38 +22,47 @@
 
 from deepchecks import vision
 
-__all__ = ['load_dataset', 'SimpleClassificationDataset', 'SimpleClassificationData']
-
-
-def load_dataset(
-    root: str,
-    train: bool = True,
-    batch_size: int = 32,
-    num_workers: int = 0,
-    shuffle: bool = True,
-    pin_memory: bool = True,
-    object_type: Literal['VisionData', 'DataLoader'] = 'DataLoader',
-    **kwargs
-) -> t.Union[DataLoader, vision.ClassificationData]:
+__all__ = ['classification_dataset_from_directory', 'SimpleClassificationDataset', 'SimpleClassificationData']
+
+
+def classification_dataset_from_directory(
```

Comment:
I think we should name it image_dataset_from_directory to match the names in Keras and Pytorch, that's what I'd look for as a user

Reply:
Thing is that it only works for classification so I am worried it will be confusing